checks/remotesettings/push_timestamp.py

```diff
@@ -48,15 +48,17 @@ async def get_remotesettings_timestamp(uri) -> str:
     return str(matched[0]["last_modified"])
 
 
-async def run(remotesettings_server: str, push_server: str) -> CheckResult:
+async def run(
+    remotesettings_server: str, push_server: str, lag_margin: int = 600
+) -> CheckResult:
     rs_timestamp = await get_remotesettings_timestamp(remotesettings_server)
     push_timestamp = await get_push_timestamp(push_server)
 
     rs_datetime = utcfromtimestamp(rs_timestamp)
     push_datetime = utcfromtimestamp(push_timestamp)
 
     return (
-        push_timestamp == rs_timestamp,
+        0 <= (rs_datetime - push_datetime).seconds < lag_margin,
```

Comment:
I don't think this is what we want. The difference between these two timestamps can become arbitrarily large – it just can't _remain_ arbitrarily large for longer than `lag_margin`. So what we want is that either `rs_datetime >= datetime.now() - lag_time` or `rs_datetime == push_datetime`.

Reply:
Oh, of course. It took me some time to understand the difference, but yes of course. Thank you for the feedback